library/src/main/java/com/google/android/exoplayer2/extractor/mkv/MatroskaExtractor.java

```diff
@@ -1403,6 +1405,7 @@ public void initializeOutput(ExtractorOutput output, int trackId) throws ParserE
           mimeType = MimeTypes.AUDIO_AAC;
           initializationData = Collections.singletonList(codecPrivate);
           break;
+        case CODEC_ID_MP2:
```

Comment:
The constant came from the platform [MP3Extractor.cpp](https://android.googlesource.com/platform/frameworks/av/+/master/media/libstagefright/MP3Extractor.cpp#431). Should be safe to use it here too as per the comment (and we might even be able to use a smaller value, without rounding to the next power of two, but I'm not sure this is a good idea).

Unrelated to this change, we could set `MP3_MAX_INPUT_SIZE = MpegAudioHeader.MAX_FRAME_SIZE_BYTES` to avoid having the magic number.